frontend/src/modules/app/UserActivity.tsx

```diff
@@ -1,9 +1,25 @@
+/**
+ * @summary Tracks User Activity
```

Comment:
What kinds of user actions count as "user activity"? Is it just mouse events, or other things too?

Reply:
It tracks on following events
['load', 'mousemove', 'mousedown', 'click', 'scroll', 'keypress']